src/ColorContextMenu-Core.package/Color.extension/class/ccmHue.saturation.brightness.alpha..st

```diff
@@ -1,4 +1,4 @@
 *ColorContextMenu-Core-instance creation
 ccmHue: hue saturation: saturation brightness: brightness alpha: alpha
 
-	^ self h: hue * 360 s: saturation v: brightness alpha: alpha
\ No newline at end of file
+	^ self h: hue * 360 s: (saturation max: 0.005) v: (brightness max: 0.005) alpha: alpha
```

Comment:
Is this a hack to preserve hue always? If yes, please move this change into `ColorChooserMorph` and add a small comment about it :)